viewer/src/public/RevealManagerBase.ts

```diff
@@ -282,16 +282,12 @@ export class RevealManagerBase {
     pipeline
       .pipe(
         auditTime(100),
-        fromThreeCameraConfig(),
+        map(camera => camera.clone()),
```

Comment:
Why have this here instead of inside the determine sector?

Reply:
To ensure that implementors of `SectorCuller` (which can be customised by the end-user) doesn't forget to clone.